metadata/static/spark/overview.md

```diff
@@ -0,0 +1,27 @@
+## Overview
+Apache Spark is a fast and general-purpose cluster computing system for big data. 
+It provides high-level APIs in Scala, Java, Python, and R, and an optimized engine that supports general computation graphs for data analysis. 
+It also supports a rich set of higher-level tools including: Spark SQL for SQL and DataFrames, MLlib for machine learning, 
+GraphX for graph processing, and Spark Streaming for stream processing.
+
+The KUDO Spark Operator creates, configures, and manages instances of Spark Operator running on Kubernetes.
+
+- Easily create and manage Spark workloads on Kubernetes using `SparkApplication` resource
+- Spark History Server support
+- Metrics collection into Prometheus via ServiceMonitors  
+- Monitor Spark jobs using pre-configured Grafana dashboards
+- High Availability deployment mode
+- Integration with external batch scheduler (Volcano)
+- Graceful rolling updates for any cluster configuration changes
+- Graceful rolling upgrades when upgrading the operator version
+- External access through LB/Nodeports
+
+## Support Level
+- Mixed workload tested with:
+  - 50 operators
+  - 1000 Spark jobs with 1000 drivers and 1000 executor pods running concurrently
+  - TeraSort benchmark successfully completed and validated  
+- Base tech support
```

Comment:
Just double checking: I just want to verify that we're for sure providing base tech support for spark?

Reply:
Not sure about this one, just copy-pasted it from another addon. Does base tech here mean Apache Spark or the Spark Operator? cc @akirillov

Reply:
I think we should remove it for now; I'll inquire with Support as to whether and when **they** can provide that.